website/content/maintainers/github/templates/required/governance-intro.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "Using the Governance Templates"
+linkTitle: "Governance Templates"
+date: 2022-05-13
+draft: true
```

Comment:
Did you intend to mark all of these with draft? They won't be visible on the site.

Reply:
If we want to get all three up at once, but fill in the other two later, I recommend putting a link to the template from these stubs and just explain that a full howto is in the works. That way the page is a bit more helpful.

Reply:
Originally we were talking about that, but at this point I don't know when the other two will be done.  As such, I want to at least get this one up for people looking at that template.

Reply:
See changed version.